docs/README.md

```diff
@@ -1 +1,116 @@
+
+Next, **add media** (either an image of a GIF) with the rendered components, so that users can better understand how the app works in practice. 
+
+![Media Placeholder](https://user-images.githubusercontent.com/52087100/71204177-42ca4f80-227e-11ea-89e6-e92e65370c69.png)
+
+## Configuration 
+
+In this section, you first must **add the primary instructions** that will allow users to use the app's blocks in their store, such as:
+
+1. Adding the app as a theme dependency in the `manifest.json` file;
+2. Declaring the app's main block in a given theme template or inside another block from the theme.
+
+Remember to add a table with all blocks exported by the app and their descriptions. You can verify an example of it on the [Search Result documentation](https://vtex.io/docs/components/all/vtex.search-result@3.56.1/). 
+
+Next, add the **props table** containing your block's props. 
+
+If the app exports more than one block, create several tables - one for each block. For example:
+
+### `block-1` props
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+
+### `block-2` props
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+Prop types are: 
+
+- `string` 
+- `enum` 
+- `number` 
+- `boolean` 
+- `object` 
+- `array` 
+
+When documenting a prop whose type is `object` or `array` another prop table will be needed. You can create it following the example below:
+
+- `propName` object:
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+
+Remember to also use this Configuration section to  **showcase any necessary disclaimer** related to the app and its blocks, such as the different behavior it may display during its configuration. 
```

Comment:
This is an instruction to clarify which info should be added to the doc - It should be replaced with real data!